search/readme.md

```diff
@@ -0,0 +1,119 @@
+# Search API
+
+## General
+Search API gives you the ability to run queries via REST (i.e. HTTP). The query language is elasticsearch Search API DSL, with certain limitations. 
+
+## License 
+Using the API requires a special license from Logz.io, and an API token which can be generated here - https://app.logz.io/#/dashboard/account/tokens 
+
+The number of queries executed is controlled and limited by Logz.io.
+
+The number of results are limited to 10,000.
+
+## Search Command
+
+### URL
+URL: https://api.logz.io/v1/search﻿
+* Make sure to use https and not http, otherwise you will get 403 Forbidden.
+
+### HTTP Headers
+* "X-USER-TOKEN" -  The value should be the API token you generated (as explained in the License section above)
+
+### Request Body 
+Elasticsearch Search API Body as documented  in [Elaticsearch documentation](https://www.elastic.co/guide/en/elasticsearch/reference/5.x/search.html), with the following restrictions:
+* The only top level elements we support in the Elasticsearch Search API are:  `query`, `from`, `size`, `sort`, `_source`, `post_filter`, `aggs`, `aggregations`
+* When using `query_string` element, you are not allowed to set its field named `allow_leading_wildcard` to true
```

Comment:
you are not allowed to set the value for the `allow_leading_wildcard` field to true